main/gcc/APKBUILD

```diff
@@ -194,6 +194,7 @@ source="ftp://gcc.gnu.org/pub/gcc/releases/gcc-${_pkgbase:-$pkgver}/gcc-${_pkgba
 	300-main-gcc-add-musl-s390x-dynamic-linker.patch
 
 	fix-rs6000-pie.patch
+	gcc-ppc-pure64.patch
```

Comment:
Could you just append these changes to 'gcc-pure64.patch' instead of adding a new patch. We did the same for s390x and aarch64. See commits f41e7533051039a224a302fafe8356548af0c543 and db1d2c069e66b3bf9962d76b0b48bfdcfc8d0384.